test/busted_runner.lua

```diff
@@ -4,4 +4,61 @@ local suffix = (platform and platform.sysname:lower():find'windows') and '.dll'
 package.path = deps_install_dir.."/share/lua/5.1/?.lua;"..deps_install_dir.."/share/lua/5.1/?/init.lua;"..package.path
 package.cpath = deps_install_dir.."/lib/lua/5.1/?"..suffix..";"..package.cpath;
 
+local uv = vim.loop
```

Comment:
```suggestion
local uv = vim.uv
```

(unless we want to backport)